tests/json/retryable_writes/bulkWrite.json

```diff
@@ -629,7 +629,132 @@
         "result": {
           "deletedCount": 0,
           "insertedIds": {
-            "0": 2
+            "1": 2
+          },
+          "matchedCount": 1,
+          "modifiedCount": 1,
+          "upsertedCount": 0,
+          "upsertedIds": {}
+        },
+        "collection": {
+          "data": [
+            {
+              "_id": 1,
+              "x": 12
+            },
+            {
+              "_id": 2,
+              "x": 22
+            }
+          ]
+        }
+      }
+    },
+    {
+      "description": "Single-document write following deleteMany is retried",
```

Comment:
The following two tests are from [SPEC-1005](https://jira.mongodb.org/browse/SPEC-1005).